lib/storage/data/file/DataFileStore.js

```diff
@@ -163,10 +173,8 @@ class DataFileStore {
                      * for the pod and can potentially cause the pod
                      * to be killed under memory pressure:
                      */
-                    ret = posixFadvise(fd, 0, size, 4);
-                    if (ret !== 0) {
-                        log.warning(
-                            `error fadv_dontneed ${filePath} returned ${ret}`);
+                    if (this.noCache) {
+                        releasePageCacheSync(filePath, fd, log);
```

Comment:
Do we account for the byte `size` here as was done before?

Reply:
see answer above